packages/headless/src/api/service/insight/get-interface/get-interface-response.ts

```diff
@@ -0,0 +1,88 @@
+export interface GetInsightInterfaceResponse {
```

Comment:
If possible, I think using the term `UserInterface` instead of just `Interface` may help distinguish these interfaces from the programmatical concept of interfaces.

Reply:
or `InsightInterfaceConfig`

Reply:
I like `InsightInterfaceConfig`, will use that instead.